client/components/domains/use-my-domain/utilities/transfer-domain-action.ts

```diff
@@ -78,9 +78,10 @@ export const transferDomainAction: AuthCodeValidationHandler = (
 
 		const startInboundTransferAndReload = async () => {
 			try {
```

Comment:
Can we unwrap from this `try / catch` (and even the `startInboundTransferAndReload`) and use the promise coming from `wpcom.req.get()` and its own `.then()` and `.catch()`?

Reply:
The `try/catch` and `async/await` syntax looks OK to me, converting to a promise didn't lead to any tangible improvement. In 526df0f I only removed the redundant check for `result.success`. The endpoint never returns a "successful" response with `success: false`. It's always a `WP_Error`and a non-200 HTTP code when anything fails.

Reply:
Looks much cleaner with `startInboundTransfer` FWIW 👍